ColonySimulator.Backend/BaseClassesTest.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace ColonySimulator.Backend;
+using ColonySimulator.persistence.Models;
+
```

Comment:
Wywal tego maina tutaj, wystarczy nam jeden w program.cs

Reply:
A to akurat zapomniałem usunąć